inference-engine/tests/functional/plugin/cpu/single_layer_tests/shuffle_channels.cpp

```diff
@@ -2,75 +2,84 @@
+        results << "Prc=" << inType << "_";
+        results << "Axis=" << std::to_string(axis) << "_";
+        results << "Group=" << std::to_string(group) << "_";
+        results << CPUTestsBase::getTestCaseName(cpuParams);
 
-        return result.str();
+        return results.str();
     }
 
 protected:
     void SetUp() override {
-        LayerTestsDefinitions::shuffleChannelsLayerTestParamsSet basicParamsSet;
+        InputShape shapes;
+        ElementType inType;
+        LayerTestsDefinitions::shuffleChannelsSpecificParams shuffleChannelsParams;
+        int axis, group;
         CPUSpecificParams cpuParams;
-        std::tie(basicParamsSet, cpuParams) = this->GetParam();
+        std::tie(shapes, inType, shuffleChannelsParams, cpuParams) = this->GetParam();
+        std::tie(axis, group) = shuffleChannelsParams;
 
         std::tie(inFmts, outFmts, priority, selectedType) = cpuParams;
+        if (selectedType.empty()) {
+            selectedType = getPrimitiveType();
+        }
+        selectedType = selectedType + "_" + InferenceEngine::details::convertPrecision(inType).name();
+        targetDevice = CommonTestUtils::DEVICE_CPU;
+        init_input_shapes({shapes});
 
-        LayerTestsDefinitions::shuffleChannelsSpecificParams shuffleChannelsParams;
-        std::vector<size_t> inputShape;
-        Precision netPrecision;
-        std::tie(shuffleChannelsParams, netPrecision, inPrc, outPrc, inLayout, outLayout, inputShape, targetDevice) = basicParamsSet;
-
-        int axis, group;
-        std::tie(axis, group) = shuffleChannelsParams;
-        auto ngPrc = FuncTestUtils::PrecisionUtils::convertIE2nGraphPrc(netPrecision);
-        auto params = ngraph::builder::makeParams(ngPrc, {inputShape});
-        auto paramOuts = ngraph::helpers::convert2OutputVector(
-                ngraph::helpers::castOps2Nodes<ngraph::op::Parameter>(params));
+        auto params = ngraph::builder::makeDynamicParams(inType, inputDynamicShapes);
         auto shuffleChannels = std::dynamic_pointer_cast<ngraph::opset3::ShuffleChannels>(
-                ngraph::builder::makeShuffleChannels(paramOuts[0], axis, group));
+                ngraph::builder::makeShuffleChannels(params[0], axis, group));
         shuffleChannels->get_rt_info() = getCPUInfo();
         ngraph::ResultVector results{std::make_shared<ngraph::opset3::Result>(shuffleChannels)};
         function = std::make_shared<ngraph::Function>(results, params, "shuffleChannels");
-
-        if (selectedType.empty()) {
-            selectedType = getPrimitiveType();
-        }
-        selectedType.push_back('_');
-        selectedType += netPrecision.name();
     }
 };
 
 TEST_P(ShuffleChannelsLayerCPUTest, CompareWithRefs) {
     SKIP_IF_CURRENT_TEST_IS_DISABLED()
 
-    Run();
-    CheckPluginRelatedResults(executableNetwork, "ShuffleChannels");
+    run();
+    // TODO: need to uncomment when this method will be updated
+    // CheckPluginRelatedResults(executableNetwork, "DepthToSpace");
```

Comment:
uncomment please

Reply:
Sure, thanks

Moreover I decided to uncomment it for another tests